src/app/clusters/administrator-commissioning-server/administrator-commissioning-server.cpp

```diff
@@ -193,6 +193,8 @@ bool emberAfAdministratorCommissioningClusterRevokeCommissioningCallback(
 {
     ChipLogProgress(Zcl, "Received command to close commissioning window");
 
+    Server::GetInstance().GetFailSafeContext().ForceFailSafeTimerExpiry();
```

Comment:
The spec doesn't say to do this, right?  Do we need a spec fix here?  Seems like we do.

Reply:
I think the spec could use some more concrete wording here. Right now, it's quite dependent on your interpretation of what it means to "revoke" a commissioning window and "expire" the pake verifier. The sdk takes the pake verifier expiration to include all open pase sessions, but it then leaves the failsafe in a bad state where there's a failsafe open that's not attached to any ongoing session.

I think the spec could use some more detail here, but I'm not sure that expiring the pase sessions is a bug. It does seem like a bug that the failsafe doesn't get closed off when the pase session expires and perhaps the spec needs to be more concrete here as well - if a session gets closed, perhaps the spec should be explicit that the associated failsafe gets expired.

Reply:
Thank you for filing https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6215